model/src/main/kotlin/config/DownloaderConfiguration.kt

```diff
@@ -23,6 +23,12 @@ import org.ossreviewtoolkit.model.SourceCodeOrigin
 import org.ossreviewtoolkit.spdx.getDuplicates
 
 data class DownloaderConfiguration(
+    /**
+     * Configuration of the license categories that should be considered for source code archiving. Using an empty list
+     * resolves archiving all packages.
```

Comment:
Use `[license categories][LicenseCategory]` and add the required import to provide some context what a license category is.

Reply:
The text could make it more clear that this is about archiving packages with licenses from those categories. Otherwise it can be confusing if you don't know the context, because it's not the license categories which are archived, but package source code.